packages/api/src/apps/datastore_api/README.md

```diff
@@ -0,0 +1,305 @@
+PRICE_UPDATE_RATE_S=900
+
+# Optional update rate for detecting new contracts, defaults to 60 (1 minute)
+DETECT_CONTRACTS_UPDATE_RATE_S=60
+
+# defaults to 864,000,000 ( 10 days): represents the max age a block will stay cached
+OLDEST_BLOCK_MS=1
+
+# these configure synthetic price feed. These price feeds may require paid api keys.
+cryptowatchApiKey=
+tradermadeApiKey=
+quandlApiKey=
+defipulseApiKey=
+
+# 0x price feed gets us market prices on tokens
+zrxBaseUrl=https://api.0x.org
+
+# how many days of price history we should query on startup, leave empty to disable, history will start at the time app is started
+backfillDays=
+
+# required for lsp and emp state updates. this is a valid multicall2 address on mainnet.
+MULTI_CALL_2_ADDRESS=0x5ba1e12693dc8f9c48aad8770482f4739beed696
+
+# any non null value will turn on debugging. This adds additional logs and time profiles for key calls.
+debug=1
+
+# LSP Creator addresses are used to discover deployed lsp contracts. Currently we have to manually specify old creator addresses
+lspCreatorAddresses=0x0b8de441B26E36f461b2748919ed71f50593A67b,0x60F3f5DDE708D097B7F092EFaB2E085AC0a82F42,0x31C893843685f1255A26502eaB5379A3518Aa5a9,0x9504b4ab8cd743b06074757d3B1bE3a3aF9cea10
+```
+
+## Starting
+
+Assuming all dependencies are installed and `.env` is configured:
+
+from package script:
+`yarn api`
+
+or directly:
+`npx ts-node src/start.ts api`
+
+## Usage
+
+All queries can be made to `http:localhost:${EXPRESS_PORT}` through a POST.
+Example: `curl -X POST localhost:8282/actions` will list available actions
+
+### Usage with Postman
+
+API can be tested with [postman](https://www.postman.com/) through this [link](https://www.getpostman.com/collections/e99f4e09e2443cb31ef4)
+You will need to set your environment variable `host` to your deployment of the API, example `http://localhost:8282`.
+
+## Actions
+
+Actions are RPC style function calls which take JSON parameters and return a JSON body. Actions are accessed through the POST method.
+This is a non exhaustive list of actions, for the full list see [action source](./services/actions.ts).
+
+The following actions can be called to the server with `curl -X POST localhost:8282/${action}`.
+Parameters are attached as JSON to the request body in the follow format: `"[...arguments]"`. Arguments should be attached to the body as an array.
+
+## Channels
+
+Channels are a way to group actions into related sets. Currently channels are used to separate queries
+across contract types, but they can theoretically group any action calls together. To call an action on
+a channel, use the channel name as part of the path. `curl -X POST localhost:8282/${channel}/${action}`.
+Current channels are
+
+- emp - all emp calls
+- lsp - all lsp calls
+- global - custom calls that are not contract type specific
+
+## Action Client
+
+See [../../examples/action-client.ts](Action Client Code) for an example of how to interface with the api from the client side.
+
+## Emp Api
```

Comment:
you can link to the api readme for these docs, similarly to what i did in the lsp readme